apps/src/templates/instructions/FeedbackStudentReviewState.jsx

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import i18n from '@cdo/locale';
+import {KeepWorkingBadge} from '@cdo/apps/templates/progress/BubbleBadge';
+import {ReviewStates} from '@cdo/apps/templates/types';
+import color from '@cdo/apps/util/color';
+
+class FeedbackStudentReviewState extends React.Component {
+  static propTypes = {
+    latestReviewState: PropTypes.oneOf(Object.keys(ReviewStates)),
+    isAwaitingTeacherReview: PropTypes.bool
+  };
+
+  renderCompleted() {
+    return (
+      <div style={styles.studentReviewState}>{i18n.reviewedComplete()}</div>
+    );
+  }
+
+  renderKeepWorking() {
+    return (
+      <div style={styles.studentReviewState}>
+        <KeepWorkingBadge style={styles.keepWorkingBadge} />
```

Comment:
Fwiw, I didn't expect the exclamation point in this component to be the same exclamation point that's used as a badge on the bubbles.  I think it's probably ok that we're sharing KeepWorkingBadge, but also thought I'd mention this in case it helps to keep the KeepWorkingBadge simpler to not have it be shared between these two places.

Reply:
Charlie also made a comment about BubbleBadge components, I've done a refactor based on their recommendation, which simplifies the relationship between the KeepWorking badge and how it's used on the bubble.